decidim-admin/app/forms/decidim/admin/organization_form.rb

```diff
@@ -43,7 +43,7 @@ class OrganizationForm < Form
       validates :default_locale, :reference_prefix, presence: true
       validates :default_locale, inclusion: { in: :available_locales }
       validates :admin_terms_of_use_body, translatable_presence: true
-      validates :comments_max_length, numericality: { greater_than: 0 }, if: ->(form) { form.comments_max_length.present? }
+      validates :comments_max_length, presence: true, numericality: { greater_than: 0 }, if: ->(form) { form.comments_max_length.present? }
```

Comment:
I think we can remove the if here when we are adding the presence validator.
```suggestion
      validates :comments_max_length, presence: true, numericality: { greater_than: 0 }
```